examples/simple_zonal_private/main.tf

```diff
@@ -18,11 +18,6 @@ locals {
   cluster_type = "simple-regional-private"
 }
 
-provider "google-beta" {
```

Comment:
Why are we removing provider definition here and updating with stable in the [other example](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/228/files#diff-f5c29161b97122fb4fcac300207057afR17)